content/en/docs/TrueNAS/ssh-connections.md

```diff
@@ -0,0 +1,172 @@
+  * Enter login credentials for the remote system.
+  * Choose an existing SSH Keypair or generate a new keypair for this connection.
+  * Clicking Save automatically connects to the remote FreeNAS/TrueNAS and configures the SSH keys.
+
+* [SSH Connections: Manual Setup](#creating-a-manual-connection-between-truenas-and-a-second-nas-system)
+  * Requires manually copying a local system public SSH key over to a remote system.
+    * Go to **System > SSH Keypairs** and open the keypair to be used for the connection.
+    * Highlight the entire Public Key text and copy it to the clipboard or download the public key as a text file.
+  * Log in to the remote system and follow the procedure recommended by that system’s software documentation to add the Public Key.
+    * [FreeNAS/TrueNAS](#adding-an-ssh-key-to-the-truenas-root-account)
+    * Go to **Accounts > Users** and edit the root account.
+    * Paste the Public Key text into the SSH Public Key box and click Save.
+  * Local system
+    * Go to **System > SSH Connections > ADD**
+      * Setup Method = Manual
+      * Enter the hostname or IP address of the remote system that has the public SSH key.
+      * Private Key: Select the Keypair that was used to copy the SSH key to the remote system.
+      * Click Discover Remote Host Key to initiate connection and add that key to the form.
+
+## Allowing SSH Access to TrueNAS
+
+Configuring and activating the SSH service is done on the **Services** page
+
+<img src="/images/services.png">
+<br><br>
+
+To configure SSH, disable the service and open the configuration screen.
+
+<img src="/images/services-ssh.png">
+<br><br>
+
+
+Configure the options as needed to match your network environment.
+Remote systems could require *root* access to the system, but be sure all security precautions are in place before setting that option.
+There are some additional options recommendations for the SSH service:
+
+* Add *NoneEnabled no* to disable the insecure *none* cipher.
+* Increase the *ClientAliveInterval* if SSH connections tend to drop.
+* *ClientMaxStartup* defaults to *10*. Increase this value when more concurrent SSH connections are required.
+
+Don't forget to re-enable the SSH service on the **Services** page when all the configuration changes are complete.
+
+## Restricting Command Line Users to *scp* or *sftp*
+
+This only works for users that use command line versions of *scp* and *sftp*.
+When SSH is configured, authenticated users with a user account can use *ssh* to log into the TrueNAS system over the network.
+User accounts are created by going to **Accounts > Users** and clicking ADD.
+While the SSH login defaults to the user home directory, users are able to navigate outside their home directory, which can pose a security risk.
+
+To allow users to use *scp* and *sftp* to transfer files between their local computer and their home directory on the TrueNAS system, while restricting them from logging into the system using *ssh*.
```

Comment:
This sentence might be missing something? or contain an extra word? or punctuation?